solanaj/src/main/java/org/p2p/solanaj/kits/transaction/mapper/TransactionDetailsNetworkMapper.kt

```diff
@@ -0,0 +1,77 @@
+package org.p2p.solanaj.kits.transaction.mapper
+
+import org.p2p.solanaj.kits.transaction.BurnOrMintDetails
+import org.p2p.solanaj.kits.transaction.CloseAccountDetails
+import org.p2p.solanaj.kits.transaction.CreateAccountDetails
+import org.p2p.solanaj.kits.transaction.SwapDetails
+import org.p2p.solanaj.kits.transaction.TransactionDetails
+import org.p2p.solanaj.kits.transaction.TransferDetails
+import org.p2p.solanaj.kits.transaction.UnknownDetails
+import org.p2p.solanaj.kits.transaction.network.ConfirmedTransactionRootResponse
+import org.p2p.solanaj.kits.transaction.parser.OrcaSwapInstructionParser
+import org.p2p.solanaj.kits.transaction.parser.SerumSwapInstructionParser
+import org.p2p.solanaj.utils.SolanjLogger
+
+class TransactionDetailsNetworkMapper(private val logger: SolanjLogger) {
```

Comment:
moved this mapper to solanaj module, it's this module API, not :app

Reply:
This mapper also can be an object